### PR TITLE
Add pointcloud sample type

### PR DESCRIPTION
**Pull request by ly-kolena**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: kolena/workflow/test_sample.py
Comment (gordonhart) [inline kolena/workflow/test_sample.py]: Requesting `POINT_CLOUD` to align with the [corresponding asset type](https://github.com/kolenaIO/kolena/blob/trunk/kolena/workflow/asset.py#L42)

**Review by gordonhart — APPROVED** [review]

**Comment by gordonhart** [comment]
@ly-kolena is this ready to land in today's release, or should we wait until after it's cut to merge?

**Comment by ly-kolena** [comment]
> @ly-kolena is this ready to land in today's release, or should we wait until after it's cut to merge?

If there's no reason for urgency, I'm thinking to merge it after cut so the it aligns with platform visualization support.

Let me know if that is not a concern or otherwise.

**Comment by gordonhart** [comment]
Agreed! No urgent reason to rush this into today's release.

**Pull request merged**
